redirect/client.js

```diff
@@ -0,0 +1,52 @@
+(function() {
+
+    function apply(context, template) {
+
+        /**
+         * Prevent redirect from occurring while in either the Template Editor or Campaign Editor.
+         */
+        if ((window.frameElement || {}).id === "siteEditorFrame"
+            || !context.targetPage.includes(window.location.hostname + window.location.pathname.replace(/\/$/gim, ""))) {
```

Comment:
Simplify the regex. No need for the extra modifiers. 
```suggestion
            || !context.targetPage.includes(window.location.hostname + window.location.pathname.replace(/\/$/, ""))) {
```